lib/WebpackOptionsApply.js

```diff
@@ -485,7 +485,13 @@ class WebpackOptionsApply extends OptionsApply {
 
 		if (options.performance) {
 			const SizeLimitsPlugin = require("./performance/SizeLimitsPlugin");
-			new SizeLimitsPlugin(options.performance).apply(compiler);
+			if (Array.isArray(options.performance)) {
```

Comment:
Doing it this way would cause repeated warnings:

* When assets fail multiple performance limits
* The hint: `You can limit the size of your bundles by using import()...`

Better support an array as argument in the SizeLimitsPlugin and keep this code as it was before.

Reply:
Also add a test for these cases.

Reply:
What do you have in mind for the SizeLimitsPlugin? Loop over the options array inside the tap hook?